cfgov/v1/auth_forms.py

```diff
@@ -36,14 +36,6 @@ class UserEditValidationMixin(PasswordValidationMixin):
     user_attribute = 'instance'
 
 
-class CFGOVUserCreationForm(UserEditValidationMixin, UserCreationForm):
-    pass
```

Comment:
@richaagarwal hmm related to the email weirdness.. this is true... lets enforce the email unique here